schemas/derivatives/SCHEMA.md

```diff
@@ -0,0 +1,69 @@
+| token_amount_usd  | (Optional) The amount of the provided token in USD.   | number |
+| liquid_token_amount | The amount of tokens supplied that are liquid and usable by users of the protocol, decimal normalized. | number |
+| liquid_token_amount_usd | (Optional) The amount of liquid tokens supplied in USD. | number |
+| total_fees_usd    | (Optional) All fees generated by the LP in the given snapshot in USD. | number |
+| user_fees_usd     | (Optional) The portion of fees accrued to users of the protocol, in USD. | number |
+| protocol_fees_usd | (Optional) The portion of fees accrued to the protocol, in USD. | number | 
+
+### Pool Snapshot
+
+| Property                | Description                                                           | Type   |
+|-------------------------|-----------------------------------------------------------------------|--------|
+| timestamp               | The timestamp of the snapshot.                                        | number |
+| chain_id                | Standard chain id.                                                    | number |
+| pool_address            | The smart contract address of the pool.                               | string |
+| funding_rate            | The funding rate in this pool at the time of the snapshot, as a percentage. | number |
+| fee_rate                | The pool's fee rate, as a percentage.                                 | number |
+| total_value_locked_usd  | (Optional) The total value locked in USD.                             | number |
+| open_interest_longs_usd | (Optional) The open interest of longs in USD.                         | number |
+| open_interest_shorts_usd| (Optional) The open interest of shorts in USD.                        | number |
+
+### Trades
+
+| Property            | Description                                         | Type   |
+|---------------------|-----------------------------------------------------|--------|
+| timestamp           | The timestamp of the trade.                         | number |
+| chain_id            | Standard chain id.                                  | number |
+| transaction_hash    | The hash of the transaction this trade was performed in. | string |
+| block_number        | The block number of the trade.                      | number |
+| pool_address        | The address of the pool this token was traded in.   | string |
+| maker_address       | The address of the maker.                           | string |
+| taker_address       | The address of the taker.                           | string |
+| token_address       | The address of the token.                           | string |
+| amount              | The amount of the token traded, decimal normalized. | number |
+| amount_usd          | (Optional) The trade amount in USD.                 | number |
+| pnl_usd             | (Optional) The total profit and loss of the trade in USD.      | number |
```

Comment:
how do we define trades here?

for example, if user does the following :
open position , increase position , decrease position , increase position then fully close position.

will this table have 5 records or just 1 record?

Reply:
Wdyt? I would imagine we would want a row in the table for each update, right?

Reply:
yes, it be nice to have all. But it would be accounted as such in most cases I know:
- open long $100
- open short $10 (i.e. decreasing long)
- close part of long $10 (also decreasing long)
- close long $90 or open short $90

Reply:
Other interesting data to have is:
- leverage
- liquidations

Reply:
In trades should we have a new field for the collateral value and then one for the long/short value? That way we can derive leverage?

I am not sure what to call these fields, @lylprg any ideas?

Reply:
I did update the table description and added a new field called `trade_type` to enter if it's an open, close, or update